javascript/javascript1/week2/student-manager.js

```diff
@@ -0,0 +1,40 @@
+const class07Students = [];
+function addStudentToClass(studentName)
+{
+  if (studentName==="")
+  {
+    console.log("name should not be empty");
+  }
+  else if(class07Students.includes(studentName))
+  {
+    console.log("student must not be same");
+  }
+  else if (class07Students.length<=5)
+  {
+    class07Students.push(studentName);
+  }
+  else if(studentName==="Queen") 
+  {
+    class07Students.push(studentName);
+  }
+  else
+  {
+    console.log("class is full")
+  }
+}
+function getNumberOfStudents()
+{
+  const NoOfStudents=class07Students.length;
+  return NoOfStudents;
```

Comment:
No need to define a new variable, you can directly return the value `return class07Students.length;`